tests/models/test_portfolio_version.py

```diff
@@ -37,3 +37,7 @@ def test_amount_constraints(self, portfolio):
             )
 
         assert "CHECK constraint failed" in str(e.value)
+
+    def test_deleted_at(self):
+        assert PortfolioVersion().deleted_at is None
+        assert PortfolioVersion(version=1).deleted_at is None
```

Comment:
You didn't save the model's object in DB.

```suggestion
        assert PortfolioVersion.create(version=1).deleted_at is None
```